pkg/cloudprovider/providers/fake/fake.go

```diff
@@ -322,3 +322,7 @@ func (f *FakeCloud) DeleteRoute(clusterName string, route *cloudprovider.Route)
 	delete(f.RouteMap, name)
 	return nil
 }
+
+func (c *FakeCloud) GetLabelsForVolume(pv *v1.PersistentVolume) (map[string]string, error) {
+	return map[string]string{"a": "1"}, nil
+}
```

Comment:
This should really be a generic function on the FakeCloud.  Maybe add a set function to set labels that the get just returns?  Or maybe add a means to externally define a function to get the volume labels?

Reply:
yep, wanted to get a full run and will fix that and switch off WIP in the commit and PR